docs/reference/quickstart/getting-started.asciidoc

```diff
@@ -1,51 +1,81 @@
 [[getting-started]]
-== Quick start: Add data using Elasticsearch APIs
+== Quick start: Indices, documents, and mappings
 ++++
-<titleabbrev>Basics: Add data using APIs</titleabbrev>
+<titleabbrev>Basics: Indices, documents, and mappings</titleabbrev>
 ++++
 
 In this quick start guide, you'll learn how to do the following tasks:
```

Comment:
Quickstart articles like this should still have an introductory paragraph. Introductory paragraphs can help search engines when searching for content.

Reply:
Roger that!

Reply:
because the intro paragraph will be prose version of the "in this quick start you'll learn how to do x,y,z" list, wonder if should remove that list entirely and let the "**On this page**" serve that porpoise 🐬

Reply:
add intro in https://github.com/elastic/elasticsearch/pull/113558/commits/31d8006392b2eb3e578309b43493ad420c6a99e6